File: java_work/ex0630/src/ex0630/ex03.java
package ex0630;

public class ex03 {

	public static void main(String[] args) {
		
		int num = 1;
		boolean search = false;
		while(num<100) {
			if((num %5 == 0) && (num %7 ==0)) {
				search = true; //search를 true로 바꿔야함.
				break;
			}
			num++;
		}
		if(search)
		System.out.println("num = "+num);
	}
}

File: python_work/0705/ex03.py
def main():
    print('main 시작')
    a = input("숫자를 입력하세요.") #type(a)=0 -> str(글자)
    print(type(a))
    num = int(a) # type(a) -> int로 나옴. 즉 a를 숫자로 변환해서 num에 입력해줌.
    print(type(num))
    if num > 0:
        print(num,"은 0보다 큽니다.")
    else: 
        print(num,"은 0보다 작습니다.")
    print('main 끝')


main()